translator/translate/otel/exporter/awsemf/kubernetes.go

```diff
@@ -275,6 +275,12 @@ func getControlPlaneMetricDeclarations(conf *confmap.Conf) []*awsemfexporter.Met
 					"apiserver_storage_list_duration_seconds",
 				},
 			},
+			{
+				Dimensions: [][]string{{"ClusterName", "priority_level"}, {"ClusterName"}},
```

Comment:
Its very messy to have most labels as pascal case but some of these new ones as snake case.
Can we add relabel rules on the prometheus receiver to convert these?

Reply:
Spoke offline, we have decided that we will maintain parity with prometheus format for control plane metrics